### ISSUE TITLE
Old annotations not showing up in Fly-canto

### ISSUE THREAD

**Issue by vmt25** [pombase/canto#2081]
In a given session - https://curation.pombase.org/flybase-test/curs/79ef0a0d649b4ffb - the genotype management page/tables mention existing annotations 

<img width="974" alt="Screen Shot 2019-10-21 at 13 03 23" src="https://user-images.githubusercontent.com/33721108/67203540-4159e380-f403-11e9-8687-4409fe81ebc4.png">

but I cannot seem them either via the ‘view annotations’ button or on the summary page… only newly generated annotations show up:

<img width="772" alt="Screen Shot 2019-10-21 at 13 05 01" src="https://user-images.githubusercontent.com/33721108/67203607-7108eb80-f403-11e9-9e5d-5ba9f36716cc.png">

**Comment by kimrutherford** [comment]
Hi Vitor.  Sorry about that.  The annotations are still in the database but they weren't showing up.

They were created before I changed the configuration for the "anatomy" annotation type.  In the canto_deploy.yaml it used to be called "phenotype" so the annotations in the database had type "phenotype".  When I changed the config I forgot to update the existing annotations to match so they weren't showing.  I've fixed the config file now so I think all the older annotations should be back.
